R/validations.R

```diff
@@ -148,12 +151,13 @@ validate_in <- function(x, choices, msg) {
 
 #' Validates that vector has length greater than 0
 #'
-#' @description `r lifecycle::badge("stable")`
+#' `r lifecycle::badge("stable")`
```

Comment:
Won't this push the second paragraph to `@details` section?

```suggestion
#' @description `r lifecycle::badge("stable")`
```